uui/components/tables/DataTableHeaderCell.tsx

```diff
@@ -30,28 +25,29 @@ export class DataTableHeaderCell<TItem, TId> extends React.Component<DataTableHe
         return css['font-size-14'];
     };
 
-    getColumnCaption = () => (
-        <div className={ css.tooltipWrapper }>
-            <Tooltip
-                trigger="hover"
-                placement="bottom-start"
-                renderContent={ !this.state.isDropdownOpen && this.props.column.info ? () => this.props.column.info : null }
-                color="default"
-                cx={ css.cellTooltip }
-                offset={ [-12, 12] }
-            >
+    getColumnCaption = () => {
+        const getTooltipContent = () => !this.state.isDropdownOpen && (
```

Comment:
Let's move the declaration outside getColumnCaption function

Reply:
done